tools/travis/test_openwhisk.sh

```diff
@@ -82,22 +82,42 @@ cd $TRAVIS_BUILD_DIR
 #
 cd $OPENWHISK_HOME
 
+# Build openwhisk image to keep test case code consistent with latest openwhisk core code
+./gradlew distDocker -PdockerImagePrefix=openwhisk -PdockerImageTag=latest
```

Comment:
As comment said, build openwhisk image to keep test case code consistent with latest openwhisk core code